src/main/java/org/dcsa/bkg/model/transferobjects/BookingConfirmationSummaryTO.java

```diff
@@ -1,22 +1,30 @@
 package org.dcsa.bkg.model.transferobjects;
 
 import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.Data;
 import org.dcsa.core.events.model.Location;
+import org.dcsa.core.model.ForeignKey;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+import org.springframework.data.relational.core.sql.Join;
 
 import javax.validation.constraints.Size;
 import java.time.OffsetDateTime;
 
 @Data
+@Table("shipment")
```

Comment:
I think it would be better to create a model for the shipment entity in the event-core, so it can be reused. And keep this DTO seperate.